Week01/Problem01/mfahrbach_01.py

```diff
@@ -0,0 +1 @@
+print(sum(n for n in range(1000) if n % 3 == 0 or n % 5 == 0))
```

Comment:
:+1: for concise code; however, sometimes it's better to spread out over multiple lines for readability.